CONTRIBUTING.md

```diff
@@ -0,0 +1,18 @@
+### General checklist
+
+* Make sure your code is compliant to Coding guidelines section below.
+* Git: Use `git add -p` to stage relevant parts of a changed file, instead of the entire file. This will help you club related code in a commit despite them being split across files. (*and* you can work also concurrently work on other code changes).
+* Git: Keep commit titles short and self-explanatory including the Github issue number (example: '#12') 
+* Git: Include a commit description that elaborates:
+ 	  * What code changed
+ 	  * What was added or removed
+* Testing: 
+ 	  * Make sure the fix fixes the bug in question
+ 	  * Run tests in `tests` dir with `nose tests`
```

Comment:
We should standardise on `./runtests.sh` from the project dir. Also, are there tests?